wagtail_live_debug/templates/wagtail_live_debug/dummychannel_detail.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'wagtail_live_debug/base.html' %}
+{% load static %}
+
+{% block title %} #{{ channel }} {% endblock title %}
```

Comment:
Should we put the `#` on the `__str__` method of a model? So that channel names are consistent?

Reply:
ALright